### ISSUE TITLE
Loss functions

### ISSUE THREAD

**Issue by mortonjt** [mortonjt/garfunkel#33]
We still need to explore alternatives for loss functions. Right now, we are just using [Frobieus norm](https://github.com/mortonjt/garfunkel/blob/master/deepblast/losses.py#L9)

It'll be nice we could also have
- Cross-entropy of ground-truth alignments
- Lower diagonal Frobieus norm ([page 11](https://arxiv.org/pdf/1802.03676.pdf))
